ui/src/components/session/SessionList.vue

```diff
@@ -56,7 +56,23 @@
           </template>
 
           <template v-slot:item.username="{ item }">
-            {{ item.username }}
+            <v-tooltip
+              v-if="!item.authenticated"
+              bottom
+            >
+              <template v-slot:activator="{ on, attrs }">
+                <span
+                  v-bind="attrs"
+                  v-on="on"
+                >{{ item.username }}</span>
+              </template>
+              <span v-if="!item.authenticated">Unauthorized</span>
+            </v-tooltip>
+            <template
+              v-if="item.authenticated"
+            >
+              {{ item.username }}
+            </template>
```

Comment:
Change the commit log for something more meaningful, similar to:

```
ui: Fix unauthenticated tooltip regression

The unauthenticated tooltip were not being shown when we hover
the mouse over the user icon while it should.

The issue has been spot by the eslint update and this commit fixes it.
```

also, this commit should be done before the update, if it comes in this PR.